### PR TITLE
Added Next permutation in Python

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
### Description
This PR adds Next Permutation in python[cp Folder]

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Solves #4230

### CODE REVIEW

File: Python/cp/Next_Permutation.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+def nenumbertPermutation(array):
```
Comment (HarshCasper) [inline Python/cp/Next_Permutation.py]: Add a beginner friendly description

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/cp/Next_Permutation.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+def nextPermutation(array):
```
Comment (HarshCasper) [inline Python/cp/Next_Permutation.py:1]: Add a beginner friendly description

File: Python/cp/Next_Permutation.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+def main():
+
+    #User input for both the strings
+    print('Enter values of the array')
+    array = list(map(int, input().rstrip().split()))
+    
+    print(nextPermutation(array))
+
+main()
```
Comment (HarshCasper) [inline Python/cp/Next_Permutation.py]: Enclose in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Hi @NEERAJAP2001 

Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Comment by NEERAJAP2001** [comment]
Hey, @HarshCasper Might be a small confusion.
There is an issue associated with this PR: #4230

**Pull request merged**